data-prepper-plugins/otel-trace-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -39,15 +41,18 @@
     private static final BigDecimal SEC_TO_MILLIS = new BigDecimal(1_000);
 
     private static final Logger log = LoggerFactory.getLogger(OTelTraceRawProcessor.class);
+    private final Counter spanErrorsCounter;
+    private final Counter resourceSpanErrorsCounter;
+    private final Counter totalProcessingErrorsCounter;
 
     //TODO: https://github.com/opendistro-for-elasticsearch/simple-ingest-transformation-utility-pipeline/issues/66
     public OTelTraceRawProcessor(final PluginSetting pluginSetting) {
-        this();
+        super(pluginSetting);
+        spanErrorsCounter = pluginMetrics.counter("spanProcessingErrors");
```

Comment:
Nit: I think I left on a comment in the first revision about putting these metric names into static variables